addons/mail/static/src/new/thread/message.xml

```diff
@@ -7,6 +7,9 @@
             <div t-if="!props.squashed" class="o-mail-avatar-container position-relative">
                 <img class="w-100 h-100 rounded-circle o_object_fit_cover o_redirect" t-att-class="{ 'cursor-pointer': props.hasActions }"
                     t-attf-src="/mail/channel/1/partner/{{message.authorId}}/avatar_128"/>
+                <div class="d-flex justify-content-center flex-shrink-0 position-absolute bottom-0 end-0 align-items-center rounded-circle bg-100 small o-muted">
```

Comment:
This wrapping `<div>` is always present when using `PartnerImStatus`, so makes sense to have it part of the component. That way we should not manually maintain in each context, and also makes it easier to reuse the component in some other contexts.